mod_customized/controllers.py

```diff
@@ -0,0 +1,102 @@
+"""
+mod_customized Controllers
+===================
+In this module, users can test their fork branch with customized set of regression tests
+"""
+from flask import Blueprint, g, request, redirect, url_for
+from github import GitHub, ApiError
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+from decorators import template_renderer, get_menu_entries
+from mod_auth.controllers import login_required, check_access_rights
+from mod_auth.models import Role, User
+from mod_test.models import Fork, Test, TestType, TestPlatform
+from mod_customized.forms import TestForkForm
+from mod_customized.models import TestFork
+from mod_test.controllers import get_data_for_test, TestNotFoundException
+from mod_auth.controllers import fetch_username_from_token
+from sqlalchemy import and_
+
+mod_customized = Blueprint('custom', __name__)
+
+
+@check_access_rights([Role.tester, Role.contributor, Role.admin])
```

Comment:
A check_access_rights decorator on a before_app_request is a bad idea.

Take a look at the menu config for the mod_ci module to see how you should do it instead.